app/views/base/_form_metadata.html.erb

```diff
@@ -5,13 +5,17 @@
   </div>
 <% elsif @change_set.primary_terms.is_a?(Hash) %>
   <% @change_set.primary_terms.each_pair do |label, fields| %>
-    <div class="panel panel-default">
-      <% unless label.empty? %>
-        <div class="panel-heading"><%= label %></div>
-      <% else %>
-        <div class="panel-heading">Metadata</div>
-      <% end %>
-      <%= render partial: 'form_metadata_inner', locals: { fields: fields, f: f } %>
-    </div>
-  <% end %>
+    <% unless @change_set.model.is_a?(Numismatics::Issue) %>
```

Comment:
For this overwrites in `app/views/base` I think you should be able to instead put a different of the template in `app/views/numismatics_issue`

Reply:
I will check this. thank you.